ts/Core/Chart/Chart.ts

```diff
@@ -742,11 +742,14 @@ class Chart {
             e = {
                 x,
                 y,
-                isInsidePlot: true
+                isInsidePlot: true,
+                options
             };
 
         if (!options.ignoreX) {
             const xAxis = (
+                options.axis && options.axis.isXAxis && options.axis
```

Comment:
Is this line necessary? Or maybe we can adapt the next line and make an exception for polar charts?

`series && (inverted && !chart.polar ? series.yAxis: series.xAxis)`.